.storybook/preview.tsx

```diff
@@ -1,10 +1,10 @@
-import React from "react";
+import * as React from "react";
 import "@patternfly/patternfly/patternfly.css";
 import "@patternfly/patternfly/patternfly-addons.css";
 import type { Preview } from "@storybook/react";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { initialize, mswLoader } from "msw-storybook-addon";
-import { handlers } from "@mocks/handlers";
+import { handlers } from "../client/src/mocks/handlers";
```

Comment:
Do the path change required for introducing vite ?
I'm not against it and I understand if changing the path help reorganizing but I'm trying to distinct what's absolutely required and optional changes to be added in a follow-up PR.
That would help understanding the change with vite.

Reply:
What I discovered, while doing the migration, is that the directory `.storybook` needs to have its own configuration for recognizing `@mocks/a/b/c` which I couldn't achieve. There must be a way but I couldn't find it.